jwtauth.go

```diff
@@ -88,6 +89,15 @@ func (ja *JwtAuth) Handle(paramAliases ...string) func(chi.Handler) chi.Handler
 				return
 			}
 
+			// Check expiry via "exp" claim
+			if exp, ok := token.Claims["exp"].(int64); ok {
```

Comment:
yea, I thought about this.

I can do the typecasting inside ok. We can write an IsExpired() method that accepts the token or claims map. We then switch on the type of exp value, whether float64, int64 etc..